pandas/tests/indexes/test_numeric.py

```diff
@@ -26,6 +26,45 @@ def full_like(array, value):
     return ret
 
 
+class TestIndexArithmeticWithTimedeltaScalar(object):
+
+    @pytest.mark.parametrize('index', [
+        Int64Index(range(1, 11)),
+        UInt64Index(range(1, 11)),
+        Float64Index(range(1, 11)),
+        pytest.param(RangeIndex(1, 11),
+                     marks=pytest.mark.xfail(reason='GH#19333 Unexplained '
```

Comment:
i'd rather not xfail this. if its failing then something is wrong.

Reply:
See conversation above.  Some of the brightest minds in all the land, plus me, couldn't figure out what circleCI was complaining about.

Reply:
where is a link to a failed build

Reply:
https://circleci.com/gh/pandas-dev/pandas/9527?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
what environment is this failing in exactly. is it 2.7, using a different language, what?